include/z64actor.h

```diff
@@ -15,6 +17,7 @@ struct GlobalContext;
 struct Lights;
 struct CollisionPoly;
 
+struct EnBox;
```

Comment:
```suggestion
```
I did some testing with `EnDoor` and `SubS` and I do not think this is needed. Should be able to remove `EnDoor` as well.

Reply:
Removing it produces lots of errors on unrelated files, so I think I'll keep it
```
In file included from include/global.h:6,
                 from src/boot_O2/gfxprint.c:1:
include/functions.h:823:26: error: ‘struct EnBox’ declared inside parameter list will not be visible outside of this definition or declaration [-Werror]
  823 | s32 func_800BE63C(struct EnBox* chest);
      |                          ^~~~~
```

Reply:
Hmmm... Interesting, wonder why it works for EnDoor but not Enbox? Wonder if its because EnBox* is an argument? Anyways, it seems we do need to keep it.